x-pack/plugins/ml/public/jobs/jobs_list/components/jobs_list_view/jobs_list_view.js

```diff
@@ -310,7 +337,44 @@ export class JobsListView extends Component {
       </div>
     );
   }
+
+  render() {
+    const { isRefreshing } = this.state;
+
+    return (
+      <React.Fragment>
+        <JobStatsBar
+          setUpdateJobStats={this.setUpdateJobStats}
+          unsetUpdateJobStats={this.unsetUpdateJobStats}
+        />
+        <div className="job-management">
+          <NodeAvailableWarning />
+          <header>
```

Comment:
I thought about that but I think it's probably okay to keep each of them separate (the node warning, new button, and refresh button) since they don't share anything and each of them can be tested on their own. 🤔